res/TemplateBatchFiles/SelectionAnalyses/ProtGTR_Fitter/ProteinGTRFit.bf

```diff
@@ -21,6 +21,7 @@ LoadFunctionLibrary("ProteinGTRFit_helper.ibf"); // Moved all functions from thi
 /*------------------------------------------------------------------------------*/
 
 utility.ToggleEnvVariable ("NORMALIZE_SEQUENCE_NAMES", 1);
+utility.SetEnvVariable ("OPTIMIZATION_PRECISION", 0.1); // for testing purposes.
```

Comment:
We are not sure! We are further not sure about this merge. I may have (definitely did?) do something backwards on my end, wherein I meant to merge some changes from Sergei into my fork before pushing to veg, but this happened in reverse order.